jazz-build-module/sonar-module-loader.groovy

```diff
@@ -0,0 +1,95 @@
+#!groovy?
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonOutput
+import groovy.transform.Field
+import java.util.regex.*
+
+echo "Sonar code analyzer module loaded successfully"
+
+@Field def config_loader
+@Field def service_config
+
+/**
+ * The Sonar module to run static code analysis - for JAVA, NodeJs & Python
+ *
+*/
+
+def initialize(configLoader, serviceConfig){
+	setServiceConfig(serviceConfig)
+	setConfigLoader(configLoader)
+}
+
+
+def cleanupCodeQualityReports(){
```

Comment:
You will have to merge the code from Sidd's PR [here](https://github.com/tmobile/jazz/pull/268/files#diff-ff060439517eb5ae97bb6d92ae8aad54)

Reply:
I think it will be better once merge Sidd's PR, and then I can merge my code, because I am seeing some extra variable passing and using, that we can avoid and may be able to refactor. Can we discuss this offline.